app/services/workflows/self_employed_workflow.rb

```diff
@@ -1,7 +1,7 @@
 module Workflows
   class SelfEmployedWorkflow < BaseWorkflowService
     def call
-      raise 'Not yet implemented: Check Fincancial Eligibility service currently does not handle self-employed applicants'
+      raise 'Not yet implemented: Check Fincancial Base service currently does not handle self-employed applicants'
```

Comment:
Nit pick - Financial is spelled incorrectly.
Are we renaming the application for a specific reason?

Reply:
Nope - that was a bug.  Fixed now